### ISSUE TITLE
Released zip file seems uses backslashes as path separators

### ISSUE THREAD

**Issue by oluceps** [subframe7536/Maple-font#78]
**Which should not, this may cause some difficulties to non-windows users.**

According to [.ZIP File Format Specification](https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT)

> 4.4.17 file name: (Variable)
>
>       4.4.17.1 The name of the file, with optional relative path.
>       The path stored MUST NOT contain a drive or
>       device letter, or a leading slash.  All slashes
>       MUST be forward slashes '/' as opposed to
>       backwards slashes '\' for compatibility with Amiga
>       and UNIX file systems etc.  If input came from standard
>       input, there is no file name field.  

### Info of latest release zips (6.0)
```console
> unzip -Z -l MapleMono-NF.zip
Archive:  MapleMono-NF.zip
Zip file size: 3352849 bytes, number of entries: 6
-rw----     2.0 fat   920916 b-   556607 defN 23-Jan-02 19:25 NF\MapleMono-NF-Bold.ttf
-rw----     2.0 fat   927060 b-   561839 defN 23-Jan-02 19:25 NF\MapleMono-NF-BoldItalic.ttf
-rw----     2.0 fat   927220 b-   560525 defN 23-Jan-02 19:25 NF\MapleMono-NF-Italic.ttf
-rw----     2.0 fat   921196 b-   555945 defN 23-Jan-02 19:25 NF\MapleMono-NF-Light.ttf
-rw----     2.0 fat   927156 b-   561137 defN 23-Jan-02 19:25 NF\MapleMono-NF-LightItalic.ttf
-rw----     2.0 fat   922136 b-   555992 defN 23-Jan-02 19:25 NF\MapleMono-NF-Regular.ttf
6 files, 5545684 bytes uncompressed, 3352045 bytes compressed:  39.6%
```
```console
> unzip MapleMono-NF.zip
Archive:  MapleMono-NF.zip
warning:  MapleMono-NF.zip appears to use backslashes as path separators
  inflating: NF/MapleMono-NF-Bold.ttf
  inflating: NF/MapleMono-NF-BoldItalic.ttf
  inflating: NF/MapleMono-NF-Italic.ttf
  inflating: NF/MapleMono-NF-Light.ttf
  inflating: NF/MapleMono-NF-LightItalic.ttf
  inflating: NF/MapleMono-NF-Regular.ttf
```
also see:
[this issue](https://superuser.com/questions/1382839/zip-files-expand-with-backslashes-on-linux-no-subdirectories) and https://github.com/icsharpcode/SharpZipLib/issues/617

**Comment by oluceps** [comment]
And I did have difficulties while packaging this font for Linux distros due to these backslashes.

**Comment by subframe7536** [comment]
fix in v6.1

test on debian 11
```
$ unzip MapleMono-NF.zip
Archive:  MapleMono-NF.zip
  inflating: NF/MapleMono-NF-Bold.ttf
  inflating: NF/MapleMono-NF-BoldItalic.ttf
  inflating: NF/MapleMono-NF-Italic.ttf
  inflating: NF/MapleMono-NF-Light.ttf
  inflating: NF/MapleMono-NF-LightItalic.ttf
  inflating: NF/MapleMono-NF-Regular.ttf
```
